boards/arm/sama5/giant-board/Kconfig

```diff
@@ -0,0 +1,127 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+if ARCH_BOARD_GIANT_BOARD
+
+choice
+    prompt "CPU Frequency"
+    default SAMA5D27_GIANT_BOARD_492MHZ
+
+config SAMA5D27_GIANT_BOARD_492MHZ
+    bool "492 MHz"
+
+endchoice # CPU Frequency
+
+config SAMA5D27_GIANT_BOARD_USBHOST_STACKSIZE
+    int "USB host waiter stack size"
+    default 1536 if USBHOST_HUB
+    default 1024 if !USBHOST_HUB
+    depends on USBHOST
+
+config SAMA5D27_GIANT_BOARD_USBHOST_PRIO
+    int "USB host waiter task priority"
+    default 100
+    depends on USBHOST
+
+config SAMA5_SDMMC
+    bool "enable SDMMC controller"
+    default y if SAMA5_HAVE_SDMMC
+    select SDIO_DMA
+    select SCHED_WORKQUEUE
+    select SCHED_HPWORK
+    select MMCSD
+    select MMCSD_SDIO
+    select SDIO_BLOCKSETUP
+    select ARCH_HAVE_SDIO
+    select SAMA5_SDMMC_DMA
+    ---help---
+    Enable SD Card interface SDMMC0. Selects SAMA5_SDMMC SAMA5_SDMMC0
+    SAMA5_SDMMC_DMA SDIO_DMA SCHED_WORKQUEUE SCHED_HPWORK SDIO_BLOCKSETUP
+
```

Comment:
In most usage, help text is indented by two TABs.

Reply:
@patacongo will fix